### PR TITLE
Improve the link "back to RSS feeds": set a CSS class

### PR DESCRIPTION
**Pull request by math-GH**
Changes proposed in this pull request:
The link "back to RSS feeds" has no CSS class to style it
grafik

This PR wraps the link with an additional `<div>`

Pull request checklist:

- [ ] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

### CODE REVIEW

File: app/views/auth/index.phtml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <?php $this->partial('aside_configure'); ?>
 
 <div class="post">
-	<a href="<?= _url('index', 'index') ?>"><?= _t('gen.action.back_to_rss_feeds') ?></a>
+	<div class="backToFeeds">
```
Comment (Frenzie) [inline app/views/auth/index.phtml]: You didn't really provide a rationale, but assuming it's desirable that should be something more like `back-link`, `link back` (yes, two separate class names), or something along those general lines.

Comment (math-GH) [reply app/views/auth/index.phtml]: I am not experienced in given the correct class name. What about this one?
```html
<div class="back-link-wrapper">
	<a class="link-back" href="<?= _url('index', 'index') ?>"><?= _t('gen.action.back_to_rss_feeds') ?></a>
</div>
```

Comment (Frenzie) [reply app/views/auth/index.phtml]: It's not about experience, but if you check a few lines down, for example, you see:

```html
		<div class="form-group">
			<div class="group-controls">
```

You'll want to try to blend in with what's already there.

> What about this one?

Without a rationale it's hard to say. The question remains if it should be so specific, but in any case it fits in. ;-)

Comment (math-GH) [reply app/views/auth/index.phtml]: Is this fine?
```html
<div class="post-link">
	<a class="link-back, link back" href="<?= _url('index', 'index') ?>"><?= _t('gen.action.back_to_rss_feeds') ?></a>
</div>
```

Comment (Frenzie) [reply app/views/auth/index.phtml]: No, that would be either `link-back` or `link back`. It's a slightly philosophical difference in how you choose to style things. ;-)

**Comment by Alkarex** [comment]
@mathContao Could you please fix the merge conflict?

**Comment by math-GH** [comment]
> 
> 
> @mathContao Could you please fix the merge conflict?

fixed

**Review by Frenzie — APPROVED** [review]
lgtm (didn't test)

**Pull request merged**